src/classes/rests/WheatherCallout.cls

```diff
@@ -0,0 +1,18 @@
+/**
+ * Created by Mariya on 29.11.2022.
+ */
+
+public with sharing class WheatherCallout {
```

Comment:
MINOR
Better name would be `OpenWeatherController` as this is rest controller. Also, please rename directory from `rests` to `webservices`